### PR TITLE
Feat add image results

### PR DESCRIPTION
**Pull request by haddadsim**
-Implemented the image results gotten from Unsplash showing 10 pictures in a row
-style and design have not been implemented

### CODE REVIEW

**Comment by george-haddad** [comment]
Needs api key in netlify env variable, we just got 401 errors on the preview deploy

**Comment by haddadsim** [comment]
> Needs api key in netlify env variable, we just got 401 errors on the preview deploy

Issue fixed after inputting api key in netlify

**Comment by george-haddad** [comment]
Just tested it out, it's working. Just a few comments.

1. When there are no results you should show some message like "no results" instead of just showing nothing

2. Consider a loading animation while the user waits for the images to load

3. The search button needs to be styled better, it looks like a white rectangle

4. The images are huge, it would be cool to load thumbnails of the images and link them to the full res ones

Aside from that this looks good to merge in terms of functionality;

**Review by george-haddad — APPROVED** [review]

File: src/components/SearchPage/SearchPage.jsx [hunk 1]
```diff
@@ -17,7 +19,12 @@ const SearchPage = () => {
       headers: {
         Authorization: API_KEY,
       },
-    });
+
+    })
+
```
Comment (george-haddad) [inline src/components/SearchPage/SearchPage.jsx]: There is too much whitespace between the closing of the block and the starting of `.then` close the gap

**Pull request merged**